### ISSUE TITLE
Loss Accumulation Error

### ISSUE THREAD

**Issue by danielkelshaw** [HamzaSardar/PyCoarseNet#29]
The following line will not run without error (maybe on `cpu` but not `cuda`):

https://github.com/HamzaSardar/PyCoarseNet/blob/b5b22da9e9c37e865fcdc8dd8882f005fe4ac739/src/train_model.py#L176

Use the following instead:

```python
train_mse_loss += batch_t_loss.item() * feature_vector.size(0) 
```

**Comment by danielkelshaw** [comment]
See the following line too: https://github.com/HamzaSardar/PyCoarseNet/blob/b5b22da9e9c37e865fcdc8dd8882f005fe4ac739/src/train_model.py#L194
